pkg/app/web/src/components/application-form/index.tsx

```diff
@@ -679,8 +679,8 @@ const SelectFromSuggestionsForm: FC<ApplicationFormProps> = memo(
           </Typography>
           <Divider />
           <Stepper activeStep={activeStep} orientation="vertical">
-            <Step key="Select Piped and Cloud provider" active>
-              <StepLabel>Select Piped and Cloud provider</StepLabel>
+            <Step key="Select where to deploy application" active>
+              <StepLabel>Select where to deploy application</StepLabel>
```

Comment:
I think the existing text is easier to get what do we want here 🤔  We empathize that piped is installed in a cluster somewhere, so knowing which piped should be used is knowing which cluster we want to deploy to, but it may confuse the new users. Happy to hear others' thoughts.